src/OrbitGl/LiveFunctionsController.cpp

```diff
@@ -49,22 +48,22 @@ const TextBox* ClosestTo(uint64_t point, const TextBox* box_a, const TextBox* bo
   return box_b;
 }
 
-const TextBox* SnapToClosestStart(uint64_t function_id) {
-  double min_us = GCurrentTimeGraph->GetMinTimeUs();
-  double max_us = GCurrentTimeGraph->GetMaxTimeUs();
+const TextBox* SnapToClosestStart(const OrbitApp* app, uint64_t function_id) {
```

Comment:
When is `OrbitApp` actually needed? As far as I can see a `const TimeGraph*` would be enough.

Reply:
Done.